project/database/cleanup.py

```
@@ -0,0 +1,209 @@
+
+    def get_cleanup_stats(self) -> dict:
+        """Get statistics about data that can be cleaned up."""
+        try:
+            now = datetime.now(timezone.utc)
+
+            # Count soft-deleted warnings by age
+            soft_deleted_30d = (
+                self.db.query(SecureWarning)
+                .filter(
+                    and_(
+                        SecureWarning.is_deleted == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `SecureWarning.is_deleted:` for truth checks

Replace with `SecureWarning.is_deleted`

(E712)